app/assets/stylesheets/cards.css.scss

```diff
@@ -0,0 +1,74 @@
+$flickr-pink: #ff2e92;
+$flickr-blue: #007bdc;
+
+$loader-size: 30px; 
+
+$loader-gutter: $loader-size/10;
+$loader-transform: $loader-size + $loader-gutter * 2;
+
+.loader {
+  background: white;
+  border: $loader-size * .75 solid transparent;
+  border-radius: $loader-size/2;
+  height: $loader-size;
+  margin: 0 auto 30px;
+  position: relative;
+  width: $loader-size * 2;
+}
+
+%loader-dot {
+  animation: pink 1.05s ease-in-out infinite;
```

Comment:
Color `pink` should be written in hexadecimal form as `#ffc0cb`<br>Color literals like `pink` should only be used in variable declarations; they should be referred to via variable everywhere else.